### PR TITLE
Update alerts namespace docs

### PR DESCRIPTION
**Pull request by colerogers**
- updates the alerts docs

### CODE REVIEW

**Review by TheIronDev — APPROVED** [review]

File: src/v2/providers/alerts/alerts.ts [hunk 1]
```diff
@@ -16,6 +17,7 @@ export interface FirebaseAlertData<T = any> {
 
 /**
  * A custom CloudEvent for Firebase Alerts (with custom extension attributes).
+ * @typeParam T - the data type for this alert that is wrapped in a FirebaseAlertData object.
```
Comment (egilmorez) [inline src/v2/providers/alerts/alerts.ts:20]: Suggest backticks for `FirebaseAlertData`

File: src/v2/providers/alerts/appDistribution.ts [hunk 2]
```diff
@@ -36,29 +37,53 @@ export const newTesterIosDeviceAlert = 'appDistribution.newTesterIosDevice';
...
   opts: AppDistributionOptions,
   handler: (
     event: AppDistributionEvent<NewTesterDevicePayload>
   ) => any | Promise<any>
 ): CloudFunction<AppDistributionEvent<NewTesterDevicePayload>>;
+
+/**
+ * Declares a function that can handle adding a new tester iOS device.
+ * @param appIdOrOptsOrHandler - A specific application, options, or an event-handling function.
+ * @param handler - Event handler which is run every time a new tester iOS devices is added.
```
Comment (egilmorez) [inline src/v2/providers/alerts/appDistribution.ts:84]: "devices" typo here.

File: src/v2/providers/alerts/crashlytics.ts [hunk 3]
```diff
@@ -117,6 +117,7 @@ export interface NewAnrIssuePayload {
 
 /**
  * A custom CloudEvent for Firebase Alerts (with custom extension attributes).
+ * @typeParam T - the data type for crashlytics alerts that is wrapped in a FirebaseAlertData object.
```
Comment (egilmorez) [inline src/v2/providers/alerts/crashlytics.ts:120]: Going global with this nit: suggest backticks for `FirebaseAlertData` everywhere it crops up, thanks!

**Review by egilmorez — COMMENTED** [review]
Some nits and a few global find/replace suggestions for you, thanks!

File: src/v2/providers/alerts/alerts.ts [hunk 4]
```diff
@@ -50,14 +52,18 @@ export type AlertType =
...
+  /** Scope the function to trigger on a specific application. */
   appId?: string;
 }
 
 /**
  * Declares a function that can handle Firebase Alerts from CloudEvents.
+ * @typeParam T - The type of FirebaseAlertData the function receives.
  * @param alertTypeOrOpts the alert type or Firebase Alert function configuration.
  * @param handler a function that can handle the Firebase Alert inside a CloudEvent.
+ * @returns A CloudFunction with the passed in type as an AlertEvent.
```
[the remark this answers is not in the document]
Comment (colerogers) [reply src/v2/providers/alerts/alerts.ts:66]: Changed to "A function that you can export and deploy."

**Review by egilmorez — APPROVED** [review]
LG, thanks!

**Pull request merged**